src/Lecture1.hs

```diff
@@ -62,8 +67,10 @@ sumOfSquares x y = error "TODO!"
 🕯 HINT: use the @mod@ function
 
 -}
+
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = if n > 0 then mod n 10 else mod (n * (-1)) 10
```

Comment:
That's correct! 💯 
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```